linter/whitelists/de.txt

```diff
@@ -216,6 +218,7 @@ Projects
 Pull
 Push
 Queue
+QR
```

Comment:
I guess `QR` is a general name which is also valid in all the other languages. That's a perfect example of when it should be added to `default.txt` instead of the language specific 😄

Reply:
Basically the same as you did with `OAuth` and `Token`.